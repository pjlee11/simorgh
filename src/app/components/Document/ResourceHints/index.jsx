import React, { Fragment } from 'react';

const ResourceHints = () => (
  <Fragment>
    <link
      rel="preconnect"
      href="https://ichef.bbci.co.uk"
      crossOrigin="anonymous"
    />
    <link
      rel="preconnect"
      href="https://static.bbci.co.uk"
      crossOrigin="anonymous"
    />
    <link
      rel="preconnect"
      href="https://gel.files.bbci.co.uk"
      crossOrigin="anonymous"
    />
    <link rel="dns-prefetch" href="https://ichef.bbci.co.uk" />
    <link rel="dns-prefetch" href="https://static.bbci.co.uk" />
    <link rel="dns-prefetch" href="https://gel.files.bbci.co.uk" />
  </Fragment>
);

export default ResourceHints;
