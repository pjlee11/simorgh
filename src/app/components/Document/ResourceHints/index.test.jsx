import React from 'react';
import { shouldMatchSnapshot } from '../../../helpers/tests/testHelpers';
import ResourceHints from './index';

describe('ResourceHints', () => {
  shouldMatchSnapshot('should be correct', <ResourceHints />);
});
