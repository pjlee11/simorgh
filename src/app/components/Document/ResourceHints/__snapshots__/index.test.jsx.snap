// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ResourceHints should be correct 1`] = `
Array [
  <link
    crossOrigin="anonymous"
    href="https://ichef.bbci.co.uk"
    rel="preconnect"
  />,
  <link
    crossOrigin="anonymous"
    href="https://static.bbci.co.uk"
    rel="preconnect"
  />,
  <link
    crossOrigin="anonymous"
    href="https://gel.files.bbci.co.uk"
    rel="preconnect"
  />,
  <link
    href="https://ichef.bbci.co.uk"
    rel="dns-prefetch"
  />,
  <link
    href="https://static.bbci.co.uk"
    rel="dns-prefetch"
  />,
  <link
    href="https://gel.files.bbci.co.uk"
    rel="dns-prefetch"
  />,
]
`;
